front/src/styles/FlexBox.ts

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components'
 
-type TFlexBox = {
+export type TFlexBox = {
   direction?: 'row' | 'row-reverse' | 'column' | 'column-reverse'
```

Comment:
This file is going to be replaced by FlexBox in IU

Reply:
TFlexbox is already exported in UI, so once is replaced, it should work fine

Reply:
ya lo estoy importando desde ./types